db/seeds.rb

```diff
@@ -1,6 +1,6 @@
 user = FactoryBot.create(:user)
 
-user.street_art.create!({
+user.street_arts.create!({
```

Comment:
all of the seeds we merged recently are broken because `street_art` was lacking an `s`